src/System.Private.CoreLib/shared/System/Char.cs

```diff
@@ -275,7 +275,7 @@ public static bool IsWhiteSpace(char c)
             {
                 return (IsWhiteSpaceLatin1(c));
             }
-            return CharUnicodeInfo.IsWhiteSpace(c);
+            return CheckSeparator(CharUnicodeInfo.GetUnicodeCategory(c));
```

Comment:
>CheckSeparator [](start = 19, length = 14)

why you think this is right?

Reply:
never mind, I am seeing the other changes in the other file

---
In reply to: [233266526](https://github.com/dotnet/coreclr/pull/20983#discussion_r233266526) [](ancestors = 233266526)